airbyte-integrations/connectors/source-salesforce/source_salesforce/api.py

```diff
@@ -208,9 +208,10 @@ def filter_streams(self, stream_name: str) -> bool:
 
     def get_validated_streams(self, config: Mapping[str, Any], catalog: ConfiguredAirbyteCatalog = None):
         salesforce_objects = self.describe()["sobjects"]
-        stream_names = [stream_object["name"] for stream_object in salesforce_objects]
+        stream_objects = [stream_object for stream_object in salesforce_objects if stream_object["queryable"]]
```

Comment:
can you add a log line that indicates which streams were omitted for not being queryable by the user and explicitly saying why they were omitted? this will help debug issues with disappearing streams if it happens

Reply:
Added